admin/syslog-ng/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+syslog-ng --version | head -n 1 | grep "$2"
```

Comment:
I would use:

```
syslog-ng --version | awk 'NR == 1 { print $2; }'
```

instead.

Reply:
I run tested it on router, but it shows only number 3.

```
root@turris:~# syslog-ng --version | awk 'NR == 1 { print $2; }'
3
```

But, thanks for suggestion anyway. I think I will drop to add test.sh for now and push the update. I will look at this next week.

Reply:
Sorry, wrote that too fast:

```
syslog-ng --version | awk "NR == 1 && \$0 =~ \"$2\" { print $0; }"
```